webpack/__tests__/score.spec.jsx

```diff
@@ -87,38 +87,42 @@ describe("<UnwrappedScore>", () => {
       <UnwrappedScore
         phrases={phrases.phrases}
         currentTime={1193}
+        startTime={phrases.phrases[0].startTime.value}
+        duration={2000}
         toggles={allTogglesOn}
       />
     );
-    const calculatedPhraseIndex = determineCurrentPhrase(
-      component.instance().props.currentTime,
-      component.instance().props.phrases
+    let { 1: currentPhraseIndex } = determinePhraseIndices(
```

Comment:
Does this offer some benefit that we wouldn't get from
```
let currentPhraseIndex  = determinePhraseIndices(component.instance().props)[1];
```

Reply:
It's just a style thing.  The `airbnb` style guide that this repo is using sets "prefer destructuring", so I destructured :).  There's a discussion [here](https://github.com/airbnb/javascript/issues/1791) fwiw.  To begin with I was resistant, but I think now that I'm used to it I'm sold.

One thing that's not mentioned in that discussion is that since arrays are simply objects in JS, you can use object destructuring to pick out the object that you want by index -- this is much better than `let [, currentPhraseIndex] = ...`, and in my opinion gives something of the best of both worlds.

Reply:
(I'm not 100% sure, but given that we're using the `eslint-config-airbnb` package in the repo, I think that doing it the other way, as in your example, would cause it to fail linting and be unmergable.)